src/components/SystemProfilePanel.tsx

```
@@ -0,0 +1,234 @@
+      ]);
+      if (d.ok) setDesktop(await d.json() as DesktopStatus);
+      if (p.ok) setPower(await p.json() as PowerStatus);
+      if (!d.ok && !p.ok) setError(t("systemProfile.loadFailed"));
+    } catch {
+      setError(t("systemProfile.loadFailed"));
+    } finally {
+      setLoading(false);
+    }
+  }, [t]);
+
+  useEffect(() => { void load(); }, [load]);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/components/SystemProfilePanel.tsx:103:26
  101 |   }, [t]);
  102 |
> 103 |   useEffect(() => { void load(); }, [load]);
      |                          ^^^^ Avoid calling setState() directly within an effect
  104 |
  105 |   const readError = async (res: Response, fallback: string) => {
  106 |     try {

(react-hooks/set-state-in-effect)